include/deal.II/grid/grid_refinement.h

```diff
@@ -48,6 +48,40 @@ template <class T> class Vector;
 namespace GridRefinement
 {
   /**
+   * Return a pair of double values of which the first is adjusted refinement
+   * fraction of cells and the second is adjusted coarsening fraction of cells.
+   *
+   *
+   * @arg @p current_n_cells is current cell number.
+   *
+   * @arg @p max_n_cells is specified a maximal number of cells. If current
+   * cell number is already exceeded maximal cell number, @p max_n_cells will
+   * be set to zero and @p bottom_fraction will be adjusted to reduce cell
+   * number to @ max_n_cells. If cell number is going to be exceeded only
+   * upon refinement, then refinement and coarsening fractions are going
+   * to be adjusted with a same ratio in an attempt to reach the maximum
+   * number of cells. Be aware though that through proliferation of refinement
+   * due to Triangulation::MeshSmoothing, this number is only an indicator.
+   * The default value of this argument is to impose no limit on the number
+   * of cells.
+   *
+   * @arg @p top_fraction is the requested fraction of cells to be refined.
+   *
+   * @arg @p bottom_fraction is the requested fraction of cells to be coarsened.
+   *
+   * @note Usually you do not need to call this function explicitly. Pass
+   * @p max_n_cells to function refine_and_coarsen_fixed_number or function
```

Comment:
I think that if you use refine_coarsen_fixed_number() and refine_and_coarsen_fixed_fraction(), they will appear correctly in the doxygen.